packages/table/src/useLeafyGreenTable/useLeafyGreenTable.types.ts

```diff
@@ -81,5 +81,14 @@ export interface LeafyGreenTable<T extends LGRowData>
    * Whether the table will have selectable rows.
    */
   hasSelectableRows: boolean;
+
+  /**
+   * Available [properties and methods](https://tanstack.com/virtual/latest/docs/api/virtualizer#virtualizer-instance) return from the Virtualizer instance.
+   */
   virtual: never;
+
+  /**
+   * Whether the rows should be memoized.
+   */
+  shouldMemoizeRows: boolean;
```

Comment:
Since this is the user-facing type, can you add more TSDoc here explaining what happens when this is true, and false